cmd/microcloud/ask.go

```diff
@@ -589,17 +671,82 @@ func (c *CmdControl) askNetwork(sh *service.Handler, systems map[string]InitSyst
 		return nil
 	}
 
-	// Environments without OVN get a basic fan setup.
-	if sh.Services[types.MicroOVN] == nil {
```

Comment:
So I think around here you'll need to check if MicroCeph is installed and only go through the new implementation if it is installed.

Reply:
fixed